mmv1/third_party/terraform/utils/test_utils.go

```diff
@@ -74,7 +74,7 @@ func (d *ResourceDataMock) Timeout(key string) time.Duration {
 type ResourceDiffMock struct {
 	Before     map[string]interface{}
 	After      map[string]interface{}
-	Cleared    map[string]struct{}
+	Cleared    map[string]interface{}
```

Comment:
I'm surprised this wasn't used anywhere... Could this be `map[string]bool` instead?